prepare/cards/almostEvilML_qa_by_lang.py

```diff
@@ -0,0 +1,38 @@
+from src.unitxt.blocks import FormTask, LoadHF, RenameFields, SplitRandomMix, TaskCard
+from src.unitxt.catalog import add_to_catalog
+from src.unitxt.operators import Apply, FilterByValues, RemoveFields
+from src.unitxt.test_utils.card import test_card
+
+langs = ["en", "de", "it", "fr", "es", "ru", "nl", "pt"]
+# Counter({'de': 2302, 'it': 2210, 'fr': 2156, 'es': 2090, 'ru': 2058, 'nl': 2017, 'en': 1995, 'pt': 1994})
+
+
+def extract_lang_from_METADATA(md_as_string: str) -> str:
+    import json
+
+    md_as_json = json.loads(md_as_string)
+    return md_as_json["language"]
+
+
+for lang in langs:
+
+    card = TaskCard(
+        loader=LoadHF(path="0x22almostEvil/multilingual-wikihow-qa-16k"),
+        preprocess_steps=[
+            Apply("METADATA", function=extract_lang_from_METADATA, to_field="extracted_language"),
```

Comment:
I think this should work:
Apply("METADATA", function=json.load, to_field="metadata")
CopyFields(field_to_field=[("metadata/language", "extracted_language")], use_query=True),

Reply:
Note the dateset has a legal disclaimer, so we will not want to use it, until approved.  At the minimum, if we add the card, we should add a link to the disclaimer.

Reply:
Thank you so much, @yoavkatz , for yet another lesson I learned from you!
Indeed all automatic tests now pass !!

what disclaimer did you have in mind?  this one?
https://www.wikihow.com/wikiHow:Creative-Commons

and how to add it to the card? manually?

Reply:
Hi @yoavkatz , Thanks again for the Apply trick and education. Indeed all automatic tests now pass !!

Please advise re. the disclaimer:
(1) What disclaimer did you have in mind? this one? https://www.wikihow.com/wikiHow:Creative-Commons
(2) and how to add it to the card? manually?